spec/authentication.yaml

```
@@ -117,6 +104,7 @@ components:
           description: "Setting the Accept-Encoding header to base64 will return a pre-encoded access token"
           schema:
             type: "string"
+            default: "base64"
```

I'm actually not sure this is accurate - there is no default on the server / the default is "text/plain", right? otherwise to add this would mean adding a backwards incompatible change

we _want_ people to use `base64` here, and we should probably at least list it as an option (can we list it as recommended?). but it is not, in fact, the default

---

@telday what do you think about this?

---

[Docs](https://docs.conjur.org/Latest/en/Content/Developer/Conjur_API_Authenticate.htm?tocpath=Developer%7CREST%C2%A0APIs%7C_____4#Request) cite `application/json` as default. Can use an `enum` to list options, but I think the description is the only way to communicate our recommendation.

```suggestion
            enum:
              - "application/json"
              - "base64"
            default: "application/json"
```